app/assets/javascripts/discourse/app/components/sidebar.hbs

```diff
@@ -6,6 +6,4 @@
       <Sidebar::Anonymous::Sections />
     {{/if}}
   </div>
-
-  <Sidebar::Footer />
```

Comment:
This would remove the footer from the anonymous sidebar and I'm not sure if it is intentional?